Modules/Core/Common/include/itkBitCast.h

```diff
@@ -19,15 +19,14 @@
 #ifndef itkBitCast_h
```

Comment:
In commit message, ODL -> ODR.

Reply:
Thanks @dzenanz  That's the One Abbreviation Rule, I guess  😺